### ISSUE TITLE
Series in library not appearing in library

### ISSUE THREAD

**Issue by Uncle-Squishy** [tachiyomiorg/tachiyomi#9488]
### Steps to reproduce

I have random series in my library that refuse to show up. 
I ran an update to my library and a series timed out, which isn't generally a big deal. Iwent to go refresh that series manually and Icouldn't find it. Idid a global search and it came up in there as already in my library. Iwent bak to double check and sure enough, it's still not showing in the library.

Ican't really explain steps to reproduce it because it is a random occurrence so far. I'm just trying to figure out what the cause is and how to prevent it in the future.

I've tried removing the series and re-adding it and that didn't work.

### Expected behavior

All of my series to actually show up in my library.

### Actual behavior

Random series aren't showing in the library.

### Tachiyomi version

App version: 0.14.6

### Android version

Android 13

### Device

Galaxy Tab S8 Ultra

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Uncle-Squishy** [comment]
Update: I tried moving it to a different category and now it shows up in that category (it's the only thing in it though) but even when I'm sitting there looking at it, if Itry to search for it, it still shows no results.

**Comment by arkon** [comment]
Do you have filters enabled?

**Comment by Squishy-Senpai** [comment]
Nope, absolutely no filters enabled. I just experienced it with another series. I wonder if it's somehow tied to another bug where it will display the English title even though it's listed on the source under the Japanese title or vice versa? Though searching for alternate titles isn't bringing up the series either. Only when I go into global search does it find it, labeled as already in my library.
